crates/cdk-cln/src/lib.rs

```diff
@@ -196,16 +197,7 @@ impl MintLightning for Cln {
         &self,
         melt_quote_request: &MeltQuoteBolt11Request,
     ) -> Result<PaymentQuoteResponse, Self::Err> {
-        let invoice_amount_msat = melt_quote_request
-            .request
-            .amount_milli_satoshis()
-            .ok_or(Error::UnknownInvoiceAmount)?;
-
-        let amount = to_unit(
-            invoice_amount_msat,
-            &CurrencyUnit::Msat,
-            &melt_quote_request.unit,
-        )?;
+        let amount = melt_quote_request.amount()?;
```

Comment:
why are not using `to_unit` and also does this mean by default we `melt_quote_request.amount()?` unit is `CurrencyUnit::Msat`